bazel/repositories.bzl

```diff
@@ -135,9 +135,9 @@ def data_dependency():
     http_archive(
         name = "openssl",
         build_file = "//bazel/thirdparty:openssl.BUILD",
-        sha256 = "7375e6200ccd1540245a39c01bc8e37750d9aad5f747ad10a168a520e95dba43",
-        strip_prefix = "openssl-9cff14fd97814baf8a9a07d8447960a64d616ada",
-        url = "https://github.com/openssl/openssl/archive/9cff14fd97814baf8a9a07d8447960a64d616ada.tar.gz",
+        sha256 = "83e45faac780fc54fad5572e2cea2ef1f92154a99c6ee74d6a7f138cb50de8b4",
+        strip_prefix = "openssl-c523121f902fde2929909dc7f76b13ceb4961efe",
+        url = "https://github.com/openssl/openssl/archive/c523121f902fde2929909dc7f76b13ceb4961efe.tar.gz",
     )
 
     http_archive(
```

Comment:
Why not also the fips one?

Reply:
fips is locked at a specific version that is fips certified